<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectCfg_Compiler</class>
 <widget class="QWidget" name="ProjectCfg_Compiler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="2">
    <widget class="QCheckBox" name="chckStringList">
     <property name="text">
      <string>String List</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QCheckBox" name="chckSRecFile">
     <property name="text">
      <string>SRec File</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="chckDbgFile">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Debug File</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Main File</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="chckMain">
     <property name="text">
      <string>Use Main File</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QComboBox" name="cmbMainFile"/>
   </item>
   <item row="6" column="3">
    <widget class="QCheckBox" name="chckRawHexFile">
     <property name="text">
      <string>RawHex File</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="chckLstFile">
     <property name="text">
      <string>List File</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="chckHexFile">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Hex File</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="chckSymbolTbl">
     <property name="text">
      <string>Symbol Table</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QCheckBox" name="chckBinFile">
     <property name="text">
      <string>Bin File</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="chckVerilogFile">
     <property name="text">
      <string>Verilog File</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="chckVHDLFile">
     <property name="text">
      <string>VHDL File</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QCheckBox" name="chckMemFile">
     <property name="text">
      <string>Mem File</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="chckMacroTbl">
     <property name="text">
      <string>Macro Table</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QCheckBox" name="chckCodeTree">
     <property name="text">
      <string>Code Tree</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
